.gitignore

```diff
@@ -135,5 +135,6 @@ shell/jina-wizard.sh
 .idea
 
 # tests binaries
-/tests/**/*.bin
-*.bin
+**/**/*.bin.ids
```

Comment:
this being needed is a smell that we are not properly using `tmpdir` in tests, where does this happen?

Reply:
the DocIDCache doesn't store the bin files properly when initiated on its own (not part of a flow)

Reply:
why not? are u passing `test_metas`?

Reply:
Hm I don't really remember. I will have a look at it in the upcoming PR where I'm adding the drivers for delete and update to DocCache

Reply:
Nvm, found the problem and fixed it